### ISSUE TITLE
Cloud Model Management: Upload from local fails with 404

### ISSUE THREAD

**Issue by maekawataiki** [awslabs/stable-diffusion-aws-extension#681]
### Describe the bug

Uploading checkpoint from local using Cloud Model Management result in error.

The error message is

```
Failed to load resource: the server responded with a status of 404 (Not Found) /checkpoints:1 
txt2img-config.js?1712760452.5336232:241 Error at txt2img-config.js?1712760452.5336232:218:27
```

The request is sent to `<ip address of webui>/checkpoints` instead of API gateway.

Seems javascript in this [file](https://github.com/awslabs/stable-diffusion-aws-extension/blob/625d69b611cfb3a5dd0cabd47d86ccca21b7b435/javascript/txt2img-config.js#L182) is not sticking with recent changes.

### Expected Behavior

Checkpoint uploaded to S3.

### Current Behavior

Uploading checkpoint result in error.

### Reproduction Steps

Stable Diffusion AWS Extension: be899370706cf005d0b5e771eaf3cf0f151ab47c
Front-end & Middleware API Version: 1.5.0

1. In Cloud Models Management, upload checkpoint from local computer.

### Possible Solution

1. Fix code in this [file](https://github.com/awslabs/stable-diffusion-aws-extension/blob/625d69b611cfb3a5dd0cabd47d86ccca21b7b435/javascript/txt2img-config.js#L182)
    1. Update id to stick with recent change.
    2. The api parameter is not using username in header, may not compatible with v1.5.

### Solution Version

v1.5.0-65d8fa8	

### AWS Region. e.g., us-east-1

us-east-1

**Comment by elonniu** [comment]
it is suggested upload model first the way of URL / civitai helper

**Comment by maekawataiki** [comment]
Yes, I've uploaded model to public s3 bucket and pass URL as temporary workaround.
